src/courtroom.cpp

```diff
@@ -2692,8 +2697,11 @@ void Courtroom::chat_tick()
       ui_vp_player_char->play_idle(m_chatmessage[CHAR_NAME],
                                    m_chatmessage[EMOTE]);
     }
-    QString f_char = m_chatmessage[CHAR_NAME];
-    QString f_custom_theme = ao_app->get_chat(f_char);
+    QString f_char;
+    QString f_custom_theme;
+    if (ao_app->is_customchat_enabled())
+      f_char = m_chatmessage[CHAR_NAME];
+      f_custom_theme = ao_app->get_chat(f_char);
```

Comment:
hey what is this? then block with no braces?

Reply:
oh fuck